src/raster/r.damflood/SWE.c

```diff
@@ -225,12 +242,12 @@ void shallow_water(double **m_h1,double **m_u1, double **m_v1, float **m_z,float
 					m_h2[row][col]=0;
 				}
 
-			} // fine continuita' a valle (IF check)
+			} // fine continuita' a valle (IF check) (end continuity downstream)
 
 
 			if (method==1 || method==2){
 				//*******************************************************************
-				// calcolo portata Q uscente dal lago solo nel caso di Hp stramazzo
+                // Calculation of flow rate Q coming out of the lake only in the case of Hp weir
```

Comment:
At what concept does `stramazzo` or `Hp stramazzo` correspond? Dam, weir, overflow, (other)? Weir isn't used in the manual page. If there is a better word, there are a couple places to be adjusted.

I assume `Hp` was a mathematical symbol.

Reply:
stramazzo = spillway

Hp is hypothesis abbr.